src/pages/alert.html

```diff
@@ -7,14 +7,19 @@
   unsubscribe-link-href="http://www.google.com"
 ></header>
 
-<alertblock 
+<alertblock
   title="20% OFF ALL LISTED ITEMS"
   text="Use our Topshop student discount code at the checkout to enjoy 10% off tops, dresses & accessories"
   button-href="http://www.google.com"
   button-text="Some button text"
   image="http://placehold.it/640x320"
 ></alertblock>
 
+<brandedappfooter
+  image="http://i.imgur.com/3xtT1vN.png"
```

Comment:
For sure. It's a realistic placeholder, but I even deliberately made the teal off-colour so it's obvious it needs to have a proper one provided and have spoken to @chrisweston92 about it